profiler/src/main/java/com/splunk/opentelemetry/profiler/TLABProcessor.java

```diff
@@ -86,13 +88,26 @@ public void accept(RecordedEvent event) {
         LogDataBuilder.create(resource, INSTRUMENTATION_LIBRARY_INFO)
             .setName(PROFILING_SOURCE)
             .setEpoch(time)
-            .setBody(stack)
+            .setBody(threadLine + "\n" + "   java.lang.Thread.State: UNKNOWN\n" + stack)
             .setAttributes(attributes)
             .build();
 
     logsProcessor.log(logData);
   }
 
+  private String buildThreadLine(RecordedThread thread) {
+    if (thread == null) {
+      return buildThreadLine("unknown", 0);
+    }
+    return buildThreadLine(thread.getJavaName(), thread.getJavaThreadId());
+  }
+
+  private String buildThreadLine(String name, long id) {
+    String sb =
+        "\"" + name + "\"" + " #" + id + " prio=0 os_prio=0 cpu=0ms elapsed=0s tid=0 nid=0 unknown";
```

Comment:
could you just omit `prio=0 os_prio=0 cpu=0ms elapsed=0s tid=0 nid=0 unknown` In my opinion it is best not to send these if we can't provide real values. If this is required by gdi spec then consider relaxing it.

Reply:
I've created https://github.com/signalfx/gdi-specification/pull/162 to address this.